.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install node packages
+        run: npm install
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Stylelint
+        run: npm run stylelint
+
+      - name: Unit test
+        run: npm run test
+
+      - name: UI test
+        uses: cypress-io/github-action@v2
+        with:
+          start: npm start
```

Comment:
How about adding a `wait-on: http://localhost:3000` here?

Reply:
Nice catch. Fixed in 0ee3b1889fb15e55515cd052742ff44ce133b8b5.